optimum/neuron/utils/hub_neuronx_cache.py

```diff
@@ -151,6 +169,33 @@ def download_file(self, filename: str, dst_path: str):
             os.symlink(local_path, dst_path)
             logger.info(f"Fetched cached {rel_filename} from {self.repo_id}")
 
+    def download_folder(self, folder_path: str, dst_path: str):
+        # Always prioritize the default cache for faster retrieval
+        if self.default_cache.exists(folder_path):
+            # cached locally
+            return True
+        else:
+            rel_folder_path = self._rel_path(folder_path)
+            try:
+                folder_info = list(self.api.list_repo_tree(self.repo_id, rel_folder_path))
+                folder_exists = len(folder_info) > 1
+            except Exception as e:
+                logger.info(f"{rel_folder_path} not found in {self.repo_id}: {e} \nThe model will be recompiled.")
```

Comment:
It should be log if `folder_exists` is `False`, not only when it fails?

Reply:
No, here it could be that the model has never been cached so unfound, I only want report when there is an issue of reading the repo (like access, wrong repo_id etc.) but not warn when it's just empty.